README.md

```diff
@@ -197,3 +207,16 @@ Object with the default options for the notifications
 
 ## defaultOptionsByType
 Object with the default options for the notifications for specific types
+
+## Building
+
+_OS X users, ensure you have `wget` installed (available in `brew`)_
+
+To run the tests:
+
+```bash
+$ ./workaround.sh
```

Comment:
@hiddentao Did you need the workaround for running locally as well?
On my machine it runs fine, the workaround was only ever ment to be run on travis